rust/actyx/ax-aql/src/language/workflow.rs

```diff
@@ -235,6 +236,14 @@ fn r_event(p: P<'_>) -> Result<WorkflowStep<'_>, anyhow::Error> {
     let _at = p.next().unwrap();
     let participant = Span::make(p.next().unwrap(), |mut p| Ok(Ident(p.non_empty_string()?)))?;
     let binders = p.map(r_binding).collect::<Result<Vec<_>, _>>()?;
+    for binding in &binders {
+        if !matches!(binding.value(), SimpleExpr::Indexing(_)) {
```

Comment:
shouldn't it also check whether it is `Ind { head, .. }` where head is `SimpleExpr::Var("_")`?